Willow.xcodeproj/project.pbxproj

```diff
@@ -969,7 +979,6 @@
 				PUBLIC_HEADERS_FOLDER_PATH = "$(CONTENTS_FOLDER_PATH)/Headers";
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 3.2;
```

Comment:
All the red throughout the Xcode project files is due to removing overrides that were the same as the project defaults.